src/index.html

```diff
@@ -5,8 +5,406 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo" href="#home">
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo__img"
+       >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="link"
+              href="#laptops_&_computers"
+              data-qa="nav-hover">
+                Laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="link" href="#gadgets">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="link" href="#tablets">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="link" href="#photo">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content">
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE_A1419_iMac_27"
+          class="card__image"
+        >
+
+        <h3 class="card__description">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          Reviews: 5
+        </div>
+
+        <div class="card__price">
+          Price:
+
+          <p class="card__value">
+            $2,199
+          </p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover">
+          buy
+        </a>
```

Comment:
apply code styling recommendations from checklist
```suggestion
        <a
          href="#"
          class="card__button"
          data-qa="card-hover"
        >
          buy
        </a>
```